src/libMessage/ZilliqaMessage.proto

```diff
@@ -351,6 +351,7 @@ message LookupSetSeedPeers
 message LookupGetDSInfoFromSeed
 {
     required uint32 listenport = 1;
+    required bool initialds = 2;
```

Comment:
Please align the number

Reply:
Thanks, I will address this immediately.